src/lib/stream/stream_cipher.cpp

```diff
@@ -24,7 +24,7 @@
   #include <botan/ofb.h>
 #endif
 
-#if defined(BOTAN_HAS_RC4)
+#if defined(BOTAN_HAS_RC4) && !defined(OPENSSL_NO_RC4)
```

Comment:
I agree on giving package maintainers explicit control would be nice.
May I close this PR and request [641](https://github.com/randombit/botan/pull/641) ?